test-runner-frontend/src/Views/Dashboard.js

```diff
@@ -112,19 +99,20 @@ const reducer = (state, action) => {
 export default function Dashboard() {
   const [state, dispatch] = useReducer(reducer, initialState);
 
+  console.log(useContext(FirestoreContext));
```

Comment:
cheeky console.log that we should remove

Reply:
Done.